content/guides/basic-setup/index.md

```diff
@@ -13,48 +13,55 @@ If you require further help, please visit our forums or ask in our Discord's hel
 
 ## How to dump your PS2 BIOS
 
-Dumping your PS2 BIOS requires a mod to your PS2 to allow arbitrary code execution.
+Dumping your PS2 BIOS is conceptually a two-step process:
+1. "Hack" your PS2 to allow arbitrary code execution
+2. Now you can run a "BIOS dumper" utility program on your PS2 that reads its BIOS and writes it to a USB drive.
 
-**Popular options:**
+There is a generally useful program, uLaunchELF, that lets you browse memory cards, DVDs, and USB drives connected to a PS and run programs from them. So in most of the approaches below, you "hack" your PS2 so that it runs uLaunchELF, and uLaunchELF in turn it lets you run the BIOS dumper utility from a USB drive.
+
+### Popular "hack" options
 
 1. FreeMcBoot Memory Card
-    - Works for all but the newest (9xxxx serial number with a date code larger than 8B) slim PS2s. Can be found online for ~10 USD. Minimal effort.
+    - Works for all but the newest (9xxxx serial number with a date code larger than 8B) slim PS2s. Can be found online for ~10 USD. Minimal effort. This boots into the uLaunchELF program.
```

Comment:
> This boots into the uLaunchELF program.

This is irrelevant, above you stated the modding methods are so you can use uLaunchELF, the list is for those possible modding methods.